tests/devices/test_default_mixed.py

```diff
@@ -265,11 +278,15 @@ def test_unitary_kraus(self, ops, tol):
     diagonal_ops = [
         (PauliZ(wires=0), np.array([1, -1])),
         (CZ(wires=[0, 1]), np.array([1, 1, 1, -1])),
+        (
```

Comment:
Not sure if we'd like to have a case here. The test seems to refer to diagonal unitary operations.

Reply:
Ok, I'll remove it. I wasn't sure either.